### ISSUE TITLE
[idea] The possibility to create network devices and assign the known hosts

### ISSUE THREAD

**Issue by leiweibau** [jokob-sk/Pi.Alert#53]
For more details you can have a look at my issue. I could also imagine that it might be a bit too much. That's why I'm still running it as "experimental".
If you think there might be interest, I'll put some work into the translation.

https://github.com/leiweibau/Pi.Alert/issues/13

**Comment by jokob-sk** [comment]
I personally like it - we could ping some people who we interacted with/opened issues to see if there is interest :) Do you think it won't be too much overhead maintaining it?

**Comment by leiweibau** [comment]
The maintaining should be limited, since I work almost exclusively with data that is already there. With a little revision of the input masks, it might be possible to eliminate one or two questions.

**Comment by jokob-sk** [comment]
Sure, then let's do it then :) 

I had also a look around at some graphical JS libraries which could be used as well, if you are interested (but I would keep it simple at first):

https://demo.jsplumbtoolkit.com/community/dynamic-anchors/ 
http://fabricjs.com/stickman  ~0.3Mb
https://docs.anychart.com/Basic_Charts/Network_Graph ~1Mb

**Comment by leiweibau** [comment]
These graphs are cool, but add a lot of complexity again. Users then push them individually and then complain when the status is not saved. For this reason I decided to use simple tabs with tables.

I came up with this idea because when I looked at my switch, I always had to search for what was connected to which port (the switch interface is a mess). These graphics are only of limited help. A table is much clearer.
